eng/pipelines/checkout-task.yml

```diff
@@ -1,5 +1,15 @@
-# Shallow checkout sources on Windows
+# Shallow checkout sources
 steps:
-  - checkout: self
-    clean: true
-    fetchDepth: 1 # Fetch only one commit
\ No newline at end of file
+  - checkout: none
+  - script: |
+      git init
+      git config --local core.compression 0
```

Comment:
Is there specific documentation which lead you to specifying these options?

Reply:
I will check but I don't think `core.compression` and `core.looseCompression` do anything. Those just mean "don't waste time compressing the git repo structures". `--no-auto-maintenance` on fetch should do the same thing.

The remaining options being set to `0` mean "use as many threads as there are CPU cores"

The documentation for all of these options can be found [here](https://git-scm.com/docs/git-config#_variables)